src/test/java/seedu/address/logic/parser/person/SetUserCommandParserTest.java

```diff
@@ -0,0 +1,32 @@
+package seedu.address.logic.parser.person;
+
+import static seedu.address.commons.core.Messages.MESSAGE_INVALID_COMMAND_FORMAT;
+import static seedu.address.logic.parser.CommandParserTestUtil.assertParseFailure;
+import static seedu.address.logic.parser.CommandParserTestUtil.assertParseSuccess;
+import static seedu.address.testutil.TypicalIndexes.INDEX_FIRST_PERSON;
+
+import org.junit.jupiter.api.Test;
+
+import seedu.address.logic.commands.person.SetUserCommand;
+
+/**
+ * As we are only doing white-box testing, our test cases do not cover path variations
+ * outside of the SetUserCommand code. For example, inputs "1" and "1 abc" take the
+ * same path through the SetUserCommand, and therefore we test only one of them.
+ * The path variation for those two cases occur inside the ParserUtil, and
+ * therefore should be covered by the ParserUtilTest.
+ */
```

Comment:
is this one necessary? Cos if yes I need to add this to ViewScheduleCommandParserTest.java as well lol

Reply:
To be honest I copy pasted this from `DeleteCommandParserTest`, it should not be necessary, I will remove.